### PR TITLE
Fix other repo issue.

### PR DESCRIPTION
**Pull request by ZePan110**
## Description

Fix other repo issue.
## Issues

List the issue or RFC link this PR is working on. If there is no such link, please mark it as `n/a`.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [ ] Others (enhancement, documentation, validation, etc.)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

**Review by Zhenzhong1 — APPROVED** [review]

**Review by lvliang-intel — APPROVED** [review]

**Comment by dbkinder** [comment]
@lvliang-intel What's the purpose of this empty GenAIExamples directory created in the GenAIExamples repo? Will it contain any documentation?

**Comment by dbkinder** [comment]
@lvliang-intel @ZePan110  What's the purpose of this empty GenAIExamples directory created in the GenAIExamples repo? Will it contain any documentation?

**Comment by ZePan110** [comment]
@dbkinder  Thanks for the heads up, I removed it in https://github.com/opea-project/GenAIExamples/pull/840.

File: GenAIExamples
Comment (lianhao) [inline GenAIExamples:1]: what's this is used for?

Comment (ZePan110) [reply GenAIExamples:1]: I removed it in https://github.com/opea-project/GenAIExamples/pull/840.

**Pull request merged**